package.json

```diff
@@ -54,7 +55,7 @@
     "zone.js": "0.8.4"
   },
   "devDependencies": {
-    "@angular/cli": "1.2.8",
+    "@angular/cli": "1.4.7",
```

Comment:
This change is probably what's making the CI build fail, is this update needed for the `--disable-host-check` flag?

Should be able to reproduce the error locally via `yarn ng build -- --aot --prod`:

```
ERROR in ./src/polyfills.ts
Module build failed: Error: AotPlugin was detected but it was an instance of the wrong class.
This likely means you have several @ngtools/webpack packages installed. You can check this with `npm ls @ngtools/webpack`, and then remove the extra copies.
    at Object.ngcLoader (/workspace/node_modules/@ngtools/webpack/src/loader.js:358:19)
 @ multi ./src/polyfills.ts
ERROR in ./src/main.ts
Module build failed: Error: AotPlugin was detected but it was an instance of the wrong class.
This likely means you have several @ngtools/webpack packages installed. You can check this with `npm ls @ngtools/webpack`, and then remove the extra copies.
    at Object.ngcLoader (/workspace/node_modules/@ngtools/webpack/src/loader.js:358:19)
 @ multi ./src/main.ts
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/document.field.detail.component.ts (240,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/document.field.detail.component.ts (285,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/namespace.list.ts (137,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/document.definition.component.ts (248,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/mapping/mapping.detail.component.ts (285,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/mapping/mapping.detail.component.ts (301,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/mapping/transition.selection.component.ts (109,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/toolbar.component.ts (164,9): Type 'this' has no properties in common with type 'Component'.
ERROR in /workspace/node_modules/syndesis.data.mapper/src/app/lib/syndesis-data-mapper/components/namespace.list.ts (164,9): Type 'this' has no properties in common with type 'Component'.
```

Reply:
If you want, I can look at opening a separate PR that updates angular-ci again and sort out whatever dependency changes would be needed for it to work.

Reply:
yeah, in fact `start:minishift` seems to still work if I revert this line locally, I'd recommend setting this version back to 1.2.8 for this PR, am planning on updating dependencies across the board this sprint and it's non-trivial :-)

Reply:
The --disable-host-check is needed to make connecting from minishift into the dev server work as the hostname won't be 127.0.0.1 or localhost.  

Doing a normal test bringing up the browser against localhost:4200 is not sufficient.  You have to use ${machine_ip}:4200 to make sure that --disable-host-check is working.

--disable-host-check did not work for me at the angular cli 1.2.8.

Reply:
actually it did not work at 1.0.3 but does at 1.2.8 
updating PR.